packages/sanity/src/structure/comments/plugin/field/CommentsFieldButton.tsx

```diff
@@ -185,9 +189,17 @@ export function CommentsFieldButton(props: CommentsFieldButtonProps) {
   }
 
   return (
-    <Tooltip portal placement="top" content={`View comment${count > 1 ? 's' : ''}`}>
+    <Tooltip
+      portal
+      placement="top"
```

Comment:
There is actually native support for singular/plural built into the i18n library. Here is an example: https://github.com/sanity-io/sanity/blob/ab0dbe1b29c766a3a771ca3c066b9af00a0cd52e/packages/sanity/src/core/form/inputs/files/common/DropMessage.tsx#L31

Reply:
Following the underscore example as provided in our docs for this, just overlooked this, thanks for pointing out 👍